docs/dao-setup.md

```diff
@@ -16,6 +16,8 @@ regtest=1
 # The default rpcPort for regtest from Bitcoin Core 0.16 and higher is: 18443
 # The default rpcPort for testnet is: 18332
 # For mainnet: 8332
+[regtest]
```

Comment:
Just wondering why this was added only to the Linux example and not the others? It should be fine to add it, but should be consistent. But not sure it is even necessary in the first place since the first line is regtest=1.

Reply:
without this i could not run in ubuntu. but i have not tested in other linux distros though.

Reply:
pls let me know whether to keep or remove this.

Reply:
@ripcurlx @wiz @Emzy Could you check in your linux or other distros to confirm this option.

Reply:
on further testing, this [regtest] needed only when we set rpcport.

Reply:
exactly. here on arch linux. if I want to set regtest-specific options, I have to use the '[regtest]' keyword to specify the config area.

Reply:
I guess we should add it for the Windows and macOS examples as well.

Reply:
could anyone confirm for atleast macOS?